01-main/packages/code-insiders

```diff
@@ -0,0 +1,8 @@
+DEFVER=1
+ASC_KEY_URL="https://packages.microsoft.com/keys/microsoft.asc"
+APT_LIST_NAME="vscode-beta"
```

Comment:
This must be `"APT_LIST_NAME=code"` or it will potentially lead to having 2 apt list entries with clashing signing key details, breaking both.
However, please also see  https://github.com/wimpysworld/deb-get/blob/main/01-main/CONTRIBUTING.md regarding nightly and beta releases.

Reply:
Are dots or dashes supported as i could use either to separate it from the stable version.
code-insiders or code.insiders

Reply:
I have just read CONTRIBUTING.md, is this still the case if it's clearly labeled as being a beta version?

Reply:
> I have just read CONTRIBUTING.md, is this still the case if it's clearly labeled as being a beta version?

I expect it would be rejected because it is labelled beta.  It _might_ be arguable that `code-insiders` is not `beta` so providing the option as well as the stable would be helpful.  Indeed, since they are in the same repository it would be good to have `code-insiders` identified because otherwise if a user installed `code` with `deb-get` they can `apt install code-insiders` but if they then `deb-get remove code` and the apt repo and key will be removed, leaving them in the lurch. So I'd say the case for an exception is strong.